.github/actions/expose-jwt-action/README.md

```diff
@@ -0,0 +1,34 @@
+# View GitHub JWT Token
+
+This action takes an optional input for the `audience` value and outputs a GitHub signed JWT. 
+
+## Inputs
```

Comment:
Do we need a plural form here?

Reply:
It's from example README: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#creating-a-readme